src/message/fetchActions.js

```diff
@@ -280,8 +256,12 @@ const fetchTopMostNarrow = () => async (dispatch: Dispatch, getState: GetState)
  * and a mainly server-side perspective:
  *   https://zulip.readthedocs.io/en/latest/subsystems/events-system.html
  *
- * Also fetch some messages eagerly, and do some miscellaneous other work
- * we want to do when starting up, or regaining a network connection.
+ * Also do some miscellaneous other work we want to do when starting
+ * up, or regaining a network connection. Mostly, we don't fetch
+ * messages here; that's dispatched from the UI component responsible
+ * for showing the messages and error states about the fetch. The
```

Comment:
Probably best to mention one or both examples -- that way the reader who's interested in understanding how we maintain the messages data can go look there to follow along.